torch/csrc/Module.cpp

```diff
@@ -606,6 +607,23 @@ PyObject* THModule_getCppBacktrace(PyObject* _unused, PyObject* args) {
   END_HANDLE_TH_ERRORS
 }
 
+// Set Accelerator Python Errors.
+// Only used when privateuse1 is registered.
+static void setAcceleratorPyError() {
+  auto device_type = at::getAccelerator(true);
```

Comment:
At initialization time here, privateuse1 is most likely not registered yet. So this will fail?

Reply:
This is only called inside the `THModule_rename_privateuse1_backend` func. At initialization time, this func is not called, It is only called when the privateuse1 try to register. And the `setAcceleratorPyError` will be called just after `c10::register_privateuse1_backend`, so I think this is ok.